tests/pytests/unit/modules/test_deb_postgres.py

```diff
@@ -0,0 +1,162 @@
+import logging
+
+import pytest
+
+import salt.modules.deb_postgres as deb_postgres
+from tests.support.mock import Mock, patch
+
+log = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.skip_unless_on_linux(reason="Only supported on Linux family"),
+]
+
+LSCLUSTER = """\
+8.4 main 5432 online postgres /srv/8.4/main \
+        /var/log/postgresql/postgresql-8.4-main.log
+9.1 main 5433 online postgres /srv/9.1/main \
+        /var/log/postgresql/postgresql-9.1-main.log
+"""
```

Comment:
I think we don't want global variables in tests... perhaps @s0undt3ch could elaborate... or tell me I'm wrong

Reply:
response from @Ch3LL and @s0undt3ch is **meh**, but doing it, need to see if I can get allowance for pytest-lazy-fixture which makes doing this so much easier